src/AWSAPI.jl

```diff
@@ -9,18 +9,15 @@
 # Copyright OC Technology Pty Ltd 2017 - All rights reserved
 #==============================================================================#
 
-
 module AWSAPI
 
-
 using AWSMetadata
 using HTML2MD
 using DataStructures
 using JSON
 
-
 sdk_module_name(service_name) = "AWSSDK.$(service_name)"
-
+is_rest_service(service) = occursin(r"^rest", service["metadata"]["protocol"])
```

Comment:
Since this is a really simple regex we can just use `startswith` instead:
```julia
julia> using BenchmarkTools, Random

julia> failure = randstring(100)
"6jm5HqlFv1ct5xgTOIswsFSEhfrgPsl6zDf3jujeHb6Nz5x5deGq3EgyTaLBqLGB9mFLIhFIAIxLc8YEeOGKF2UVL3RKxzz8WCXM"

julia> succeed = "rest" * failure
"rest6jm5HqlFv1ct5xgTOIswsFSEhfrgPsl6zDf3jujeHb6Nz5x5deGq3EgyTaLBqLGB9mFLIhFIAIxLc8YEeOGKF2UVL3RKxzz8WCXM"

julia> @btime occursin(r"^rest", $failure)
  23.209 ns (0 allocations: 0 bytes)
false

julia> @btime occursin(r"^rest", $succeed)
  24.115 ns (0 allocations: 0 bytes)
true

julia> @btime startswith($failure, "rest")
  4.030 ns (0 allocations: 0 bytes)
false

julia> @btime startswith($succeed, "rest")
  8.556 ns (0 allocations: 0 bytes)
true
```